packages/mobile/src/identity/matchmaking.test.ts

```diff
@@ -63,19 +74,27 @@ describe('Fetch contact matches', () => {
       [mockE164Number2]: { contactId: 'contactId2' },
     }
 
+    const state = createMockStore({
+      web3: { account: mockAccount },
+      account: { pincodeType: PincodeType.CustomPin },
+    }).getState()
+
     await expectSaga(fetchContactMatches, e164NumberToRecipients)
       .provide([
         [call(getConnectedUnlockedAccount), mockAccount],
         [call(getUserSelfPhoneHashDetails), phoneHashDetails],
+        [matchers.call.fn(isAccountUpToDate), true],
+        [matchers.call.fn(PNPUtils.Matchmaking.getContactMatches), [mockE164Number2]],
       ])
+      .withState(state)
       .put(addContactsMatches(expectedMatches))
       .run()
   })
 })
 
-describe(obfuscateNumberForMatchmaking, () => {
+describe(PNPUtils.Matchmaking.obfuscateNumberForMatchmaking, () => {
   it('Hashes sigs correctly', () => {
```

Comment:
Would be good to move this test into contract kit. Seems odd to test it from the outside like this.

Reply:
Since we need to get this out soon, I'm going to postpone this to a future PR. I've created this item to track it here:
https://github.com/celo-org/celo-monorepo/issues/4592
Thanks!